jina/clients/__init__.py

```diff
@@ -36,7 +36,7 @@ def Client(
     """Create a Client. Client is how user interact with Flow
 
     :param asyncio: If set, then the input and output of this Client work in an asynchronous manner.
-    :param host: The host address of the runtime, by default it is 0.0.0.0.
+    :param host: The host address of the runtime, by default it is 0.0.0.0. In the case of an external Executor (`--external` or `external=True`) this can be a list of hosts, separated by commas. Then, every resulting address will be considered as one replica of the Executor.
```

Comment:
is the new description relevant to client ? or is it just auto-generated ?

Reply:
It is auto generated, due to our parser structure I don't think that there is a way to disentangle these things.